mapper/census_test.go

```diff
@@ -1367,3 +1420,12 @@ func TestCreateCensusDatasetLandingPage(t *testing.T) {
 	})
 
 }
+
+func TestCleanDimensionsLabel(t *testing.T) {
+	Convey("Removes categories count from label - case insensitive", t, func() {
+		So(cleanDimensionLabel("Example (100 categories)"), ShouldEqual, "Example")
+		So(cleanDimensionLabel("Example (7 Categories)"), ShouldEqual, "Example")
+		So(cleanDimensionLabel("Example (1 category)"), ShouldEqual, "Example")
+		So(cleanDimensionLabel("Example (1 Category)"), ShouldEqual, "Example")
```

Comment:
Can you expand the test coverage to include an empty string, string without brackets and strings with multiple brackets
e.g. 
```
So(cleanDimensionLabel(""), ShouldEqual, "") // fairly trivial but you don't have any error handling and we don't want the app to crash 
So(cleanDimensionLabel("Example 1 category"), ShouldEqual, "Example 1 category")
So(cleanDimensionLabel("Example (something in brackets) (1 Category)"), ShouldEqual, "Example (something in brackets)")
```